lib/Composer.php

```diff
@@ -100,6 +100,52 @@ public static function executeMigrationsUp(Event $event)
         }
     }
 
+
+    /**
+     * @param Event $event
+     */
+    public static function executeBundleMigrationsUp(Event $event)
+    {
+        $consoleDir = static::getConsoleDir($event, 'bundle migrations');
+
+        if (null === $consoleDir) {
+            return;
+        }
+
+        $bundles = include('var/config/extensions.php'); // path is based on composer.json's path
```

Comment:
This is not ideal, because this path can be customized. 
Maybe we should use `./bin/console pimcore:bundle:list` to get the list of enables bundles? We could introduce another flag e.g. `--json` to get the list in an easy to read format. 
What do you think?

Reply:
You are right, I added the mentioned `--json` option for `pimcore:bundle:list`.